buildSrc/src/main/kotlin/boudicca-springboot-app.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id("boudicca-kotlin")
+
+    id("org.springframework.boot")
+    id("io.spring.dependency-management")
+    kotlin("plugin.spring")
+}
+
+dependencies {
+    api("org.springframework.boot:spring-boot-starter-web")
+    api("com.fasterxml.jackson.module:jackson-module-kotlin")
+    api("org.jetbrains.kotlin:kotlin-reflect")
+    api("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.3.0")
+
+    developmentOnly("org.springframework.boot:spring-boot-devtools")
+    testImplementation("org.springframework.boot:spring-boot-starter-test")
+}
+
+task<Exec>("imageBuild") {
+    inputs.file("src/main/docker/Dockerfile")
+    inputs.files(tasks.named("bootJar")) //TODO
```

Comment:
leftover TODO

Reply:
made it more clear, this is still an open todo